extensions/vscode/src/watchers.ts

```diff
@@ -82,6 +83,31 @@ export class WatcherManager implements Disposable {
   }
 }
 
+/**
+ *  Manages file watchers for a specific Content Record
+ */
+export class ContentRecordWatcherManager implements Disposable {
```

Comment:
This is very similar to our `ConfigWatcherManager` and I figured it may expand in the future to contain more than `.contentRecord` so kept the structure the same.